pkgs/servers/etcd/3.5.nix

```
@@ -0,0 +1,36 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "etcd";
+  version = "3.5.1";
+
+  vendorSha256 = "sha256-ad89jDLLeAJizQMmO0lPH8N9GdaBrn9NyXGCxVT3fHE=";
+
+  doCheck = false;
+
+  src = fetchFromGitHub {
+    owner = "etcd-io";
+    repo = "etcd";
+    rev = "v${version}";
+    sha256 = "sha256-Ip7JAWbZBZcc8MXd+Sw05QmTs448fQXpQ5XXo6RW+Gs=";
+  };
+
+  buildPhase = ''
+    patchShebangs .
+    ./build
```

```suggestion
    ./build.sh
```
script is deprecated.